### ISSUE TITLE
lmplot hue variable with single regression line

### ISSUE THREAD

**Issue by Bercio** [mwaskom/seaborn#443]
I'd like to color with different hues the different points on the graph, but retain a single regression line.
At the moment setting hue='foo' in an lmplot draws a differrent regression line for every hue, whils i would like to retain a single one(the one you get if you don't set hue) but color the points. is it possible ?

**Comment by mwaskom** [comment]
From lmplot's perspective (actually from FacetGrid's) the scatterplot and regression line are considered one item, but you could do something like this:

```python
g = sns.lmplot(x="x", y="y", hue="g", data=df, fit_reg=False)
sns.regplot(x="x", y="y", data=df, scatter=False, ax=g.axes[0, 0])
```

This won't work with multiple column or row facets.

**Comment by Bercio** [comment]
Thanks, that's really what I was looking for. I've just used a for loop on
the variable I'd have set as col to to get around the lack of it, not as
nice but works none the less, thanks a lot


   Lorenzo

**Comment by clarelou0128** [comment]
I also used this way to generate regression plot with two colors. However, the figure size is unable to be adjusted for some reason. It remains small even when I keep increasing the figure size via "plt.figure(figsize=(20,20))". Every time, when the figure is generated, this figure size message is also printed: "<Figure size 1440x1440 with 0 Axes>". The number "1440x1440" change when the figure size is changed but not the figure itself.. I set the figure size before I do both sns.lmplot and sns.reglot. Does anyone know why?
